readme.md

```diff
@@ -1,19 +1,29 @@
 # GitHub Actions Git
 
-A collection of GitHub actions for Git projects.
+A collection of GitHub actions.
 
-## Example Playbook
+## Create or Update Git Tag
+This action will crate a new tag. If the tag already exists it will be overwritten (requires force-push).  
+The tag will be created for the commit sha that matches `${{ github.sha }}`
+
+### Example
 
 ```yaml
-  - name: Build code
-    uses: aboutbits/github-actions-git/xxx@v1
+  - name: Checkout source code
+    uses: actions/checkout@v3
+    
+  - name: Tag Deployment
+    uses: aboutbits/github-actions-git/create-or-update-git-tag@v1
```

Comment:
```suggestion
    uses: aboutbits/github-actions-git/create-or-update-tag@v1
```

I would leave the word `git` away, since we are already in that scope.